meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
Maybe we could do reverse, have logs enabled by default and disable them with a switch in `setup-mender-configuration.py` (and a corresponding env variable in docker-compose)?